### ISSUE TITLE
Question about contribution and distribution

### ISSUE THREAD

**Issue by Czajnikowski** [twostraws/Inferno#29]
Hey @twostraws 👋

I'd like to showcase one more effect in your repo, kind of a scan-glitch effect, here's a WIP/sneak peek:

https://github.com/twostraws/Inferno/assets/973682/b180b5d9-3992-4cb5-882a-dfb109da43c2

However, while it's a great idea to showcase it in your repo, it's not necessarily a good idea to keep it there from a "distribution" point of view... It's a small piece of code that ideally should be maintained independently in a separate repo. I think that it'd be a good idea to develop it in a separate Swift package and integrate it with your repo using SPM.

What do you think?

**Comment by twostraws** [comment]
That looks like a regular water ripple effect; perhaps I'm not seeing what you're referring to. However, if you want to run your own repository, you're welcome to do so; there's no need to have any involvement with Inferno.

**Comment by Czajnikowski** [comment]
> That looks like a regular water ripple effect; perhaps I'm not seeing what you're referring to.

Indeed, I didn't think about it, sorry for the confusion 🙃 I'm working on something more configurable. For example, here's the same content with a different configuration, it's a WIP:

https://github.com/twostraws/Inferno/assets/973682/e6e19374-e7c9-4edb-8b73-ac9a4eee39f9

> However, if you want to run your own repository, you're welcome to do so; there's no need to have any involvement with Inferno.

So my question was mostly about that part. I just wasn't sure how to meet two goals - "register" something with your repo to enrich the collection and allow for effortless integration with other projects. The middle ground is an actual implementation inside of your repo + a link to my repo somewhere in comments, I guess?
